src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukPagination.scala.html

```diff
@@ -14,68 +14,80 @@
  * limitations under the License.
  *@
 
+@import uk.gov.hmrc.govukfrontend.views.viewmodels.pagination.Pagination
+@import uk.gov.hmrc.hmrcfrontend.views.Utils.toClasses
+@import uk.gov.hmrc.hmrcfrontend.views.Utils.toAttributes
+
 @this()
 
-@(params: Pagination)
+@(params: Pagination)(implicit messages: Messages)
 @import params._
 
 @blockLevel = @{items.isEmpty && {next.isDefined || previous.isDefined}}
```

Comment:
best to leave it since it's what govuk use and anything else will be nightmare if we need to make changes in future, I sort of get it after squinting at design system examples "we're making them be display:block links that will stack vertically" but would much rather something like "verticallyStacked" or "largerNextPrevLinks" or something more like that 🤷‍♂️

Reply:
yeah, this is just govuk-frontend's naming convention, and I guess we don't want to diverge too much further